backend/functions/lib/use-cases/invalid-chapter.ts

```diff
@@ -0,0 +1,23 @@
+import { CaseListDbResult } from '../adapters/types/cases';
+
+namespace UseCases {
+
+  export class InvalidChapter {
+
+    returnEmptyList(): CaseListDbResult {
```

Comment:
I would say that a more meaningful name here might be something like `returnInvalidChapterResponse`